ast/src/test/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitorTest.java

```diff
@@ -165,11 +165,7 @@ public void testParseAndPrintComputeShader() throws Exception {
         + "layout(local_size_x = 128, local_size_y = 1) in;\n"
         + "void main()\n"
         + "{\n"
-        + " for(\n"
-        + "     uint d = gl_WorkGroupSize.x / 2u;\n"
-        + "     d > 0u;\n"
-        + "     d >>= 1u\n"
-        + " )\n"
+        + " for( uint d = gl_WorkGroupSize.x / 2u; d > 0u; d >>= 1u )\n"
```

Comment:
Remove two spaces - after "(" and before ")", in relation to the above comments.

Reply:
Similar fixes needed to other tests.

Reply:
At the end of this file, can you please add the following test which guards against declarations in a loop being put on the same line?

```
  @Test
  public void testParseAndPrintLoopWithDeclarationsInside() throws Exception {
    final String program = ""
        + "void main()\n"
        + "{\n"
        + " for(int i = 0; i < 10; i ++)\n"
        + "  {\n"
        + "   int a;\n"
        + "   int b;\n"
        + "  }\n"
        + "}\n";
    assertEquals(program, PrettyPrinterVisitor.prettyPrintAsString(ParseHelper.parse(program
    )));
  }
```

Reply:
Done.